src/main/java/demon/Demon.java

```diff
@@ -0,0 +1,128 @@
+package demon;
+
+import java.nio.file.NoSuchFileException;
+
+import java.io.IOException;
+
+import java.util.Scanner;
+
+import java.time.format.DateTimeParseException;
```

Comment:
Small issue, but I think you accidentally added a new line for every import. Additionally, perhaps you can try ordering them in a specific way (alphabetical) and make it consistent throughout the repo? 🤔

Reply:
Sure, thanks!